toolkit/resources/assets/grub2/grub.cfg

```diff
@@ -16,4 +16,29 @@ menuentry "CBL-Mariner" {
 	if [ -f $bootprefix/$mariner_initrd ]; then
 		initrd $bootprefix/$mariner_initrd
 	fi
-}
\ No newline at end of file
+}
+
+### BEGIN /etc/grub.d/20_linux_tboot ###
```

Comment:
As discussed offline, please move this menuentry into the readme as a template to allow users to configure tboot, but with their specific settings